src/algorithms/tracking/IterativeVertexFinder.cc

```diff
@@ -56,7 +65,8 @@ void eicrecon::IterativeVertexFinder::init(std::shared_ptr<const ActsGeometryPro
 }
 
 std::unique_ptr<edm4eic::VertexCollection> eicrecon::IterativeVertexFinder::produce(
-    std::vector<const ActsExamples::Trajectories*> trajectories) {
+    std::vector<const ActsExamples::Trajectories*> trajectories,
+    std::vector<const edm4eic::ReconstructedParticle*> reconParticles) {
```

Comment:
This is what needs changing to fix your compilation. We should not introduce new interfaces that use `std::vecotr<podio_type>`.

Reply:
addressed in the following commit